dev/index.html

```diff
@@ -53,7 +53,7 @@
                     <option value="HorizontalVertical">Horizontal-Vertical</option>
                     <option value="Horizontal">Horizontal</option>
                     <option value="Vertical">Vertical</option>
-                    <option value="Center" selected="yes">Center</option>
+                    <option value="Center" selected="selected">Center</option>
```

Comment:
Or we can put no value, like 
![image](https://user-images.githubusercontent.com/4921914/122920629-e4de6280-d361-11eb-8a7e-9a36765dc226.png)

Reply:
as you wish, as far as we remove selected="yes" which may work but which is syntactically incorrect

Reply:
done in 1467fbc